ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/sdn2ovn-rollback/provision/cucushift-installer-rehearse-aws-ipi-ovn-sdn2ovn-rollback-provision-chain.yaml

```diff
@@ -0,0 +1,11 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-ovn-sdn2ovn-rollback-provision
+  steps:
+  - chain: ipi-conf-aws
+  - ref: sdn-conf
+  - chain: ipi-install  
+  - ref: enable-qe-catalogsource
+  - ref: ovn-sdn-migration
+  - ref: ovn-sdn-migration-rollback
```

Comment:
So after rollback, the cluster still using sdn, right? If that, sounds like place the workflow in aws/ipi/sdn sounds more reasonable.

Reply:
> So after rollback, the cluster still using sdn, right? If that, placing the workflow in aws/ipi/sdn sounds more reasonable.

@zhaozhanqi How do you think this suggestion?

Reply:
I think both sdn or ovn are OK, since it's a feature for testing sdn migration to ovn and rollback.